Author: Jamie Strandboge <jamie@canonical.com>
Description: allow Google Chrome and add access to @{PROC}/[0-9]*/statm
Bug-Ubuntu: https://launchpad.net/bugs/1361372

Index: lightdm-1.11.9/data/apparmor/abstractions/lightdm_chromium-browser
===================================================================
--- lightdm-1.11.9.orig/data/apparmor/abstractions/lightdm_chromium-browser
+++ lightdm-1.11.9/data/apparmor/abstractions/lightdm_chromium-browser
@@ -15,6 +15,7 @@
   /opt/google/chrome-stable/google-chrome-stable Cx -> chromium,
   /opt/google/chrome-beta/google-chrome-beta Cx -> chromium,
   /opt/google/chrome-unstable/google-chrome-unstable Cx -> chromium,
+  /opt/google/chrome/google-chrome Cx -> chromium,
 
   # Allow ptracing processes in the chromium child profile
   ptrace peer=/usr/lib/lightdm/lightdm-guest-session//chromium,
@@ -56,6 +57,7 @@
 
     @{PROC}/[0-9]*/ r,                 # sandbox wants these
     @{PROC}/[0-9]*/fd/ r,              # sandbox wants these
+    @{PROC}/[0-9]*/statm r,            # sandbox wants these
     @{PROC}/[0-9]*/task/[0-9]*/stat r, # sandbox wants these
 
     /selinux/ r,
